### DISCUSSION TITLE
Programatically store and display values in MultiSelect component

### DISCUSSION THREAD

**Discussion by xoitx** [mantinedev/mantine#3953] [Q&A]
How do I push values that are not in the option list as created values programatically onto the multiselect input.

optionlist = ['alice','bob']

dropdown will show alice and bob, also there is option to create on typing. Once created it will display in the input field.
So if I push alice into the value state of the multiselect, it shows alice.
But ,when I push some random value, it stores in the array state. but its not displaying in the multiselect input field of the multiselect component

Note: I'm trying to put values into the value state array programatically, I'm not selecting from dropdown.

Regards

**Comment by rtivital** [comment]
In order for the value to be displayed, it must be included in data. Other options are not supported.
